feature/auth/forgot_password/build.gradle.kts

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation(project(mapOf("path" to ":data:auth")))
     implementation(project(mapOf("path" to ":core:network")))
```

Comment:
Use implementation(project(":core:network)) at the end of file, and remove this line

Reply:
In this case you already have this dependency, so just remove this line